phpunit/style-engine/class-wp-style-engine-gutenberg-test.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Tests the Style Engine class and associated functionality.
+ *
+ * @package    Gutenberg
+ * @subpackage block-library
+ */
+
+/**
+ * Tests for registering, storing and generating styles.
+ */
+class WP_Style_Engine_Gutenberg_Test extends WP_UnitTestCase {
+	function test_returns_inline_styles_from_string() {
+		$style_engine   = WP_Style_Engine_Gutenberg::get_instance();
+		$block_styles   = array(
+			'spacing' => array(
+				'padding' => '111px',
+			),
+		);
+		$padding_styles = $style_engine->get_inline_css_from_block_styles(
```

Comment:
should we try to align the naming and arguments with the JS version? 

`generate( $style_object, $style_options )`

Reply:
Ah yeah, sounds like a good idea to be consistent across the libraries, even if to avoid confusion when reading both code bases 😄 

I try for verbosity since, in my opinion `generate` doesn't really tell much of a story unless we accept the implication that we're generating a string of raw styles. Documentation will mitigate this I think. 

Furthermore, if we introduce different return values such as formatted class rules or even classnames we could use the options array argument to drive them, or just create a new method.